src/components/CollectionSharingModal/CollectionSharingModal.styl

```diff
@@ -0,0 +1,115 @@
+.CollectionSharingModal
```

Comment:
Let's have it refactor this to use curly braces like the rest of the stylus files

Also have it use our themed variables not put in it's own color codes, it messed up at least the dark themed disabled color. For the buttons, it shouldn't be coloring them anyways, it should be using the css classes like other buttons.

Reply:
Done - at least it doesn't move around now

Reply:
(I made both the input and the button stick to the top, and the open/private selector takes the same space whether visible or not, and moved over "indented" just a bit")